compute_sdk/globus_compute_sdk/sdk/batch.py

```diff
@@ -15,17 +15,20 @@ def __init__(
         endpoint_id: UUID_LIKE_T,
         task_group_id: UUID_LIKE_T | None,
         request_queue=False,
+        serialization_strategy: SerializationStrategy | None = None,
     ):
         """
         :param task_group_id: UUID of task group to which to submit the batch
         :param endpoint_id: UUID of endpoint where tasks should be executed
         :param request_queue: Whether to request a result queue from the web service;
             typically only used by the Executor
+        :param serialization_strategy: The strategy to use when serializing task
+            arguments
         """
         self.task_group_id = task_group_id
         self.endpoint_id = endpoint_id
         self.tasks: dict[str, list[str]] = defaultdict(list)
-        self._serde = ComputeSerializer()
+        self._serde = ComputeSerializer(strategy_data=serialization_strategy)
```

Comment:
Any reason not to accept the original `ComputeSerializer` object as a constructor argument rather than recreating one here?

Reply:
Only reason was to match the `Client` arguments. But on reflection, this is a more "inner" class, so no need for the interfaces to match. Updated